doc/source/ray-core/fault-tolerance.rst

```diff
@@ -0,0 +1,63 @@
+Fault tolerance
+===============
+
+Ray is a distributed system, and that means failures can happen. Generally, failures can
+be classified into two classes: 1) failures internal to Ray, and 2) application-level failures. The first can be triggered by node failures,
```

Comment:
Remove the comma. 
"1) failures internal to Ray and 2) application-level failures."